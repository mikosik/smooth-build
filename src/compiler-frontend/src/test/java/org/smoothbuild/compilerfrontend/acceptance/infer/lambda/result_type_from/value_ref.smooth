Int intValue = 7;
myValue = () -> intValue;