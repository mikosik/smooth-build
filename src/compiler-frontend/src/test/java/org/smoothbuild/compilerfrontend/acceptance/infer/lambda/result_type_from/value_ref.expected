Int intValue<>
  = 7;
()->Int myValue<>
  = () -> intValue<>;
